File: tests/fixtures/code_samples/py/sample_002.py
"""Sample 2: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_2(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 2."""
    total = 2
    for x in xs:
        total += int(x)
    return total


def operation_2_pure(value: int) -> int:
    return (value * 2) % 7919

File: tests/fixtures/code_samples/ts/sample_002.ts
/** Sample 2: small utility. */

export function operation2(xs: number[]): number {
  let total = 2;
  for (const x of xs) total += x;
  return total;
}

export function operationPure2(value: number): number {
  return (value * 2) % 7919;
}

